lib/signaling/v2/peerconnection.js

```diff
@@ -417,7 +421,7 @@ class PeerConnectionV2 extends StateMachine {
     }
     const peerConnectionState = {
       ice: {
-        candidates: this._localCandidates.slice(),
+        candidates: this._isIceLite ? [] : this._localCandidates.slice(),
         revision: this._localCandidatesRevision++,
```

Comment:
We are incrementing revision even when we are not emitting candidates event. Is that ok?